README.md

```diff
@@ -159,6 +159,59 @@ You'll need to upload these to an s3 bucket, then you can use these files to
 [create a simulation application](https://docs.aws.amazon.com/robomaker/latest/dg/create-simulation-application.html),
 and [create a simulation job](https://docs.aws.amazon.com/robomaker/latest/dg/create-simulation-job.html) in RoboMaker.
 
+
+## Generate map for your world
+
+Requires ROS-Melodic + Gazebo9.
+
+### Pre-build
+
+```bash
+# Install dependencies (Ubuntu >18.04)
+sudo apt-get install libxml-xpath-perl libxml2-utils
+chmod +x genmap.sh
```

Comment:
You can commit chmod flags to git, assuming it works in github too.

https://stackoverflow.com/questions/40978921/how-to-add-chmod-permissions-to-file-in-git

Reply:
Done..confirmed it works